cfme/scripting/setup_env.py

```diff
@@ -92,7 +92,12 @@ def setup_ha_env(cfme_version, provider_type, provider, lease, desc):
     ip2 = apps[2].hostname
     opt = '5' if cfme_version >= "5.8" else '8'
     rep = '6' if cfme_version >= "5.8" else '9'
-    mon = '9' if cfme_version >= "5.8" else '12'
+    if cfme_version < '5.8':
```

Comment:
**REQUIRED:** Please use this instead, it makes the logic of version picking more consistent to read, even if slightly less efficient.

```python
from widgetastic.utils import VersionPick, Version
mon = VersionPick({
    Version.lowest(): '12',
    '5.8': '9',
    '5.9.3': '8'
}).pick(cfme_version)
```